### PR TITLE
Warning for table max rows displayed

### LINKED ISSUE #4124: Notebooks: Surface to Users that Rows in Results May Be Limited by Cap
We need to do a better job informing users that their results may be limited by a cap (which is configurable in user settings).

Need to verify with Smitha on a final UX for this.

### PR DESCRIPTION
**Pull request by chlafreniere**
Fixes #4124.

### CODE REVIEW

File: src/sql/workbench/services/notebook/sql/sqlSessionManager.ts [hunk 1]
```diff
@@ -25,7 +25,7 @@ import * as notebookUtils from 'sql/parts/notebook/notebookUtils';
 import { IConfigurationService } from 'vs/platform/configuration/common/configuration';
 
 export const sqlKernel: string = localize('sqlKernel', 'SQL');
-export const sqlKernelError: string = localize("sqlKernelError", "SQL kernel error");
+export const sqlKernelError: string = localize('sqlKernelError', 'SQL kernel error');
```
Comment (anthonydresser) [inline src/sql/workbench/services/notebook/sql/sqlSessionManager.ts]: You should use double quotes when localizing strings `localize('sqlKernelError', "SQL kernel error")`

Comment (chlafreniere) [reply src/sql/workbench/services/notebook/sql/sqlSessionManager.ts]: Why, out of curiosity?

Comment (kevcunnane) [reply src/sql/workbench/services/notebook/sql/sqlSessionManager.ts]: Could we get a linting rule on this one? I've messed this up a lot, and looking at the code this is wildly inconsistent.

Comment (anthonydresser) [reply src/sql/workbench/services/notebook/sql/sqlSessionManager.ts]: Its just a way to distinguish between what strings are visible on the UI (should be localized) and which ones aren't.

Comment (chlafreniere) [reply src/sql/workbench/services/notebook/sql/sqlSessionManager.ts]: Roger that

**Pull request merged**